worlds/noita/__init__.py

```diff
@@ -36,21 +39,32 @@ class NoitaWorld(World):
     web = NoitaWeb()
 
     # Returned items will be sent over to the client
-    def fill_slot_data(self):
-        return {name: getattr(self.multiworld, name)[self.player].value for name in self.option_definitions}
+    def fill_slot_data(self) -> Dict[str, Any]:
+        return {
```

Comment:
```suggestion
        return self.options.as_dict("death_link", "bad_effects", etc)
```
do you really need the start inventory in your slot data? please don't send data you don't need.

Reply:
Removed that and bad effects, forgot that start_inventory_from_pool was handled with item sends rather than something the client needs, been a long day